lib/gitlab/fogbugz_import/importer.rb

```diff
@@ -0,0 +1,287 @@
+module Gitlab
+  module FogbugzImport
+    class Importer
+      attr_reader :project, :repo
+
+      def initialize(project)
+        @project = project
+
+        import_data = project.import_data.try(:data)
+        repo_data = import_data['repo'] if import_data
+        @repo = FogbugzImport::Repository.new(repo_data)
+
+        @known_labels = Set.new
+      end
+
+      def execute
+        return true unless repo.valid?
+
+        data = project.import_data.try(:data)
+
+        client = Gitlab::FogbugzImport::Client.new(token: data['fb_session']['token'], uri: data['fb_session']['uri'])
```

Comment:
Yea, I've been in contact with the gem maintainer, we are working on a solution to get a new maintainer.